math.js

```diff
@@ -14,9 +14,19 @@ function getAge() {
     return 31;
 }
 
+function getCalvin() {
+    return "AWP_M4";
+}
+
+function getDean() {
+    return "DAD?";
+}
+
 module.exports = {
     sum: sum,
     multiple: multiple,
     getName: getName,
     getAge: getAge,
+    getCalvin: getCalvin,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/dejaniskra/fake-app/pullRequest?prid=8428048)
```suggestion
    getCalvin,
```